### ISSUE TITLE
Modifies whitespace within sql strings

### ISSUE THREAD

**Issue by eki** [alekseyl/niceql#16]
Newlines and extra whitespace are collapsed into single spaces within sql string literals. When running INSERT or UPDATE statements through Niceql - as with the `pg_adapter_with_nicesql` option - can lead to data corruption.

Example:

```
[2] pry(main)> Niceql::Prettifier.prettify_sql("select 'foo\nbar  baz'")
=> "select \e[0;36;49m'foo bar baz'\e[0m"
```

I think the bug is likely here: https://github.com/alekseyl/niceql/blob/master/lib/niceql.rb#L129

I apologize that I don't have time to contribute a bug fix PR.

Without a fix, I don't think `pg_adapter_with_nicesql` can be used safely even in development.

**Comment by alekseyl** [comment]
hm, missed this issue. 

Acknowledged. Don't have ETA though, but at least now I know that there is an issue )

**Comment by alekseyl** [comment]
> Is this true even if pg_adapter_with_nicesql is turned off?

nope, its not. It's not touching SQL before execution unless specified. 

> In other words: is it the case that any usage / installation of this gem within a rails application is inherently unsafe, or is it only unsafe if you configure it certain ways?

Yes you can configure it temporarily to work in an unsafe way, to debug ceratin query, but that is not default configuraiton, and there is a WARNING about that.

**Comment by alekseyl** [comment]
Anyway, I'm started to think about removing this setting completely. It's too way dangerous if enabled in prod. Even though I'm constantly pointing out about that, even if I would fix bug, still too way dangerous to be used for all queries...

**Comment by alekseyl** [comment]
Good news everyone, finally I've managed to spend some noticeable amount of time and completely refactored core logic piece, which will bring the dollarsigned literals and also will keep literals as is: https://github.com/alekseyl/niceql/pull/21

Fix is ready to merge, I'll take a look with a fresh eye, and then it will be released.

**Comment by alekseyl** [comment]
fixed in 0.6 ver, fill free to check!
